bin/migration-scripts/0.14.203.js

```diff
@@ -0,0 +1,21 @@
+import { traversePageBlocks } from '../utils.js';
+
+export const description = 'v0.14.203';
+
+export default traversePageBlocks(adjustApplicationCustomForm);
+
+function adjustApplicationCustomForm(block) {
+  if (block?.type !== 'ApplicationFormCustom' || !block?.content?.sections) {
+    return;
+  }
+  const sections = block.content.sections;
+  sections.map((section) =>
+    section?.inputs.map((_) => {
```

Comment:
На всякий случай. Вдруг и inputs не будет

```suggestion
    section?.inputs?.map((_) => {
```